llvm/test/Transforms/InstCombine/exp2-to-ldexp.ll

```diff
@@ -0,0 +1,122 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 4
+; Note: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 4
+; RUN: opt -S -passes=instcombine %s | FileCheck %s
+; xUN: opt -mtriple=aarch64-pc-linux-gnu -S -passes=instcombine %s | FileCheck -check-prefix=AARCH64 %s
```

Comment:
xUN?

Reply:
I guess xUN is left for these FIXME tests. I saw some use of xUN in codegen tests.